index.html

```diff
@@ -45,6 +45,238 @@ <h4>i am a software developer</h4>
+
+          <li class="card-li">
+            <a href="" class="card-a">rubby</a>
+          </li>
+        </ul>
+        <button class="card-btn">see project</button>
+      </div>
+      <div class="card-body">
+        <h1 class="card-title">Profesional Art Printing Data</h1>
+        <p class="supporting-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="card-ul">
+          <li class="card-li">
+            <a href="" class="card-a">html</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">bootstrap</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">rubby</a>
+          </li>
+        </ul>
+        <button class="card-btn">see project</button>
+      </div>
+      <div class="card-body">
+        <h1 class="card-title">Profesional Art Printing Data</h1>
+        <p class="supporting-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="card-ul">
+          <li class="card-li">
+            <a href="" class="card-a">html</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">bootstrap</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">rubby</a>
+          </li>
+        </ul>
+        <button class="card-btn">see project</button>
+      </div>
+      <div class="card-body">
+        <h1 class="card-title">Profesional Art Printing Data</h1>
+        <p class="supporting-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="card-ul">
+          <li class="card-li">
+            <a href="" class="card-a">html</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">bootstrap</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">rubby</a>
+          </li>
+        </ul>
+        <button class="card-btn">see project</button>
+      </div>
+      <div class="card-body">
+        <h1 class="card-title">Profesional Art Printing Data</h1>
+        <p class="supporting-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="card-ul">
+          <li class="card-li">
+            <a href="" class="card-a">html</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">bootstrap</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">rubby</a>
+          </li>
+        </ul>
+        <button class="card-btn">see project</button>
+      </div>
```

Comment:
Hey, I think you should add some padding or margin on the left and right sides of the work section, you know, as in the Figma template of your choosing. I am sure you will like it much better, once you add some padding or margin.
This is how it looks now.
![image](https://user-images.githubusercontent.com/33268927/170989364-d4372ee7-afda-4d04-9797-91a8d5b6d316.png)